### PR TITLE
Fix bug causing plot_channel_maps(..) to break when textcolor='notext' set

### PR DESCRIPTION
**Pull request by drtobybrown**
Fixes issue #689

### CODE REVIEW

File: spectral_cube/spectral_cube.py [hunk 1]
```diff
@@ -3409,6 +3409,8 @@ def plot_channel_maps(self, nx, ny, channels, contourkwargs={}, output_file=None
                         textcolor = 'w'
                     else:
                         textcolor = 'k'
+    
+                ax.tick_params(direction='in', color=textcolor)
```
Comment (keflavich) [inline spectral_cube/spectral_cube.py:3413]:
```suggestion
                ax.tick_params(color=textcolor)
```

File: spectral_cube/spectral_cube.py [hunk 2]
```diff
@@ -3418,8 +3420,6 @@ def plot_channel_maps(self, nx, ny, channels, contourkwargs={}, output_file=None
                 ax.coords[0].set_ticklabel_position('')
                 ax.coords[1].set_ticklabel_position('')
 
-            ax.tick_params(direction='in', color=textcolor)
```
Comment (keflavich) [inline spectral_cube/spectral_cube.py:3421]: suggestion:
```
            ax.tick_params(direction='in')
```

**Review by keflavich — CHANGES_REQUESTED** [review]
small change: I think the ticks pointing in should happen either way.

But, otherwise, this looks good

**Comment by e-koch** [comment]
I think all the test failures here are a carryover from #688

**Comment by keflavich** [comment]
I'm merging this from the command line with one additional small tweak; unfortunately I can't push to this branch so I'm kinda crossing my fingers that I didn't break the build by removing whitespace and re-adding the ticks-in

**Comment by keflavich** [comment]
This has been merged despite github's complaints.

**Pull request closed without merging**